troposphere/static/js/components/modals/image_version/ImageVersionEditModal.react.js

```diff
@@ -288,9 +311,12 @@ export default React.createClass({
         ended = this.state.versionEndDate;
       }
 
-      if(!name) {
+      if(name == null) {
           return (<div className="loading"/>);
       }
+      if(images == null) {
+          return (<div className="loading" />);
+      }
```

Comment:
Would it be fair to combine these? 

```
if(name == null || images == null) {
            return (<div className="loading"/>);
}
```

I might consider a comment to capture the **intent** here because someone might come along and make this "truthy" later and not realized that there is a _strong_ intentional action take to only look for `null`. I mean, we would not expect `name` or `images` to ever be `undefined`. 

Maybe I should ask if there was an intentional decision? If so, I might include a comment.